float64 targetHeading
string pointOfSail